follower-search.js

```diff
@@ -37,7 +37,7 @@ const getFollowerList = async (userName) => {
 };
 
 //array of ids
-const getTweetHistoryOfIds = async (listOfIds, updateDBWithUserInfo, userName, db) => {
+const getTweetHistoryOfIds = async (listOfIds, userData, updateDBWithUserInfo, userName, db) => {
```

Comment:
We now introduce a new param 'userData'.
The reasoning is so that we can store all the misc follower profile data as well which will be useful for later. Might as well put them in the additionalInfo property within our document.